src/components/GrowerDetail.js

```diff
@@ -268,7 +268,7 @@ const GrowerDetail = ({ open, growerId, onClose }) => {
     setSnackbarLabel(label);
     setSnackbarOpen(true);
   }
-
+  console.log('grower', grower);
```

Comment:
Make sure to remove logging when you're done.
If you think it's best to leave some logging in for deployment, use the `loglevel` library to ensure it doesn't pollute the console in production:
```
log.debug('grower', grower);
```

Reply:
I apologize, I just removed it. I will keep this in mind for future situations.